### ISSUE TITLE
Tried Dosbox-X with Arkanoid 2- Revenge of Doh (1989)(Taito Corporation) and it doesn't work

### ISSUE THREAD

**Issue by Docwiz2** [joncampbell123/dosbox-x#3374]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar bug report(s) already exists?

- [X] I have searched and didn't find any similar issues.

### What operating system(s) this bug have occurred on?

Windows 10  (21H2)

### What version(s) of DOSBox-X have this bug?

0.83.23

### Describe the bug

Arkanoid 2- Revenge of Doh (1989)(Taito Corporation) works on other versions of DosBox, but will not play the game with DosBox-X.

You can unarchive it in a folder and you can set it up, but you cannot play it as it just stops and locks up...

I have it set-up with configuration of Adlib and VGA, it does NOT WORK IN DOSBOX-X

### Expected behavior

Using other versions of DosBox Arkanoid 2- Revenge of Doh (1989)(Taito Corporation), you can unzip the file in a folder and then run the .EXE file and set up a configuration with VGA and Adlib and the configuration file will be created.

If you try running Arkanoid 2 on any other DosBox it works great, but it will not run with DoxBox-X.

### Steps to reproduce the behaviour

1. Download the Dosbox-X from the website
2. Unzip the file
3. unzip Arkanoid 2- Revenge of Doh (1989)(Taito Corporation) to a folder
4. Run the *.exe file and create a configuration of Adlib and VGA and Mouse
5. You cannot run the game after configuration has been created.
6 

### Used configuration

```ini
I thought it was something I changed, so I unarchived the zipped Dosbox-X from the website and ran it from there with Adlib support and VGA and it had the same behavior.

So, it's nothing I did with the emulator or configuration.  It doesn't work with the default configuration.

Something has changed with Dos-Box, please investigate the issue.
```

**Comment by rderooy** [comment]
I just tried it and it worked just fine.

I'm assuming your issue is the message "Packed file is corrupt" when running the EXE?

If so, a basic google search would have told you to run "loadfix" before starting the game.

**Comment by joncampbell123** [comment]
Works here, 0.84.1

**Comment by Docwiz2** [comment]
Nope, it doesn't say that all.  It just locks up for me.  I set the file so
that it uses adlib and VGA and it just freezes.
When I use another Dosbox it just works as it should.

1) I unzipped the file into a new folder
2) I run the *.exe file and create a configuration of Adlib and VGA
3) I run the *.exe file again and it freezes

I was using the previous version of DosBox-X that I had before April and it
would just freeze the emulator.

So, what I did is I used the DosBox-X emulator directly from my download
folder and unzipped it again using the default configuration.
Did exactly the above again and it still froze.

So, I was like I will use another Dosbox (Daum version from 2015) and it
ran like a champ.

The only time I see that error "Packed file is corrupt"  is if I use 4DOS.
Again, this is why I stopped using it as well.
It doesn't run all the games (See Epix Summer Games 2 below) and it causes
problems with this packed executable stuff.

I had ran Epix Summer Games 2 using 4DOS, and it would just go back to dos,
even when I used the
fix -a command.
